tests/testapps/CTCatalog/CTCatalog/Tests/CTCAlignmentTestViewController.m

```diff
@@ -150,8 +153,9 @@ - (void)viewDidAppear:(BOOL)animated {
 }
 
```

Comment:
You also need to implement viewDidLayoutSubviews() and calls [_leftView setNeedsDisplay] for screen rotations scenarios to work and re-draw